validation/cta-1dc/run_HLI_3D.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
UTF8 is the default. No need to put this in all / any Python script. Remove this line.